src/main/java/AbstractingDBApplication.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by mowall on 15/9/16.
+ */
+public class AbstractingDBApplication {
```

Comment:
if its a actual application why have the classname as `AbstractingDBApplication`?